io.gitlab.gregorni.ASCIIImages.json

```
@@ -0,0 +1,86 @@
+{
+    "app-id" : "io.gitlab.gregorni.ASCIIImages",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
```

```suggestion
    "runtime-version" : "44",
```

44 was released today.